src/actions/index.js

```diff
@@ -0,0 +1,26 @@
+import sampleData from './sampleData';
+
+/**
+ * Makes the mock "API request"
+ */
+const fetchProducts = () => {
+  return new Promise((resolve) => {
+    window.setTimeout(() => {
+      resolve(sampleData.products);
+    }, 1000);
+  });
+}
+
+/**
+ * Retrieves product list from mock API
+ */
+export const fetchProductsFromAPI = () => {
+  return (dispatch) => {
+    return fetchProducts().then(
+      (resp) => dispatch({
+        type: 'UPDATE_PRODUCTS',
```

Comment:
It's common to define action types as constants. https://redux.js.org/basics/actions/

Can you explain why?